File: modul 6/Bahasa C/PRAK604-2410817320010-RabiahRiskaAmaliah.c
#include <stdio.h>
#include <string.h>

int main() {
    char asli[100], palsu[100];
    int kode, pesan, bintang = 0, pagar = 0;

    scanf(" %[^\n]%*c", asli);
    scanf(" %[^\n]%*c", palsu);

    kode = strlen(asli);
    pesan = strlen(palsu);

    if (kode != pesan) {
        printf("Panjang kalimat berbeda, pesan palsu\n");
        return 0;  
    }

    for (int i = 0; i < kode; i++) {
        if (asli[i] == ' ' && palsu[i] == ' ') {
            continue; 
        } else if (asli[i] == palsu[i]) {
            printf("*");
            bintang++;  
        } else {
            printf("#");
            pagar++;  
        }
    }

    printf("\n");
    printf("* = %d\n", bintang);
    printf("# = %d\n", pagar);

    if (bintang >= pagar) {printf("Pesan Asli\n");}
    else {printf("Pesan Palsu\n");}

    return 0;
}

File: github/modul 6/Bahasa py/PRAK604-2410817320010-RabiahRiskaAmaliah.py
asli = input().strip()
palsu = input().strip()
kode = len(asli)
pesan = len(palsu)
bintang = 0
pagar = 0

if kode != pesan:
    print("Panjang kalimat berbeda, Pesan Palsu")
    exit()

for i in range(kode):
    if asli[i] == palsu[i]:  
        if asli[i] == " ":  
            print(" ", end="")  
        else:
            print("*", end="")  
            bintang += 1
    else:  
        print("#", end="")
        pagar += 1
print()  

print(f"* = {bintang}")
print(f"# = {pagar}")

if bintang >= pagar:
    print("Pesan Asli")
else:
    print("Pesan Palsu")
